src/services/external-integrations/UsherIntegration.ts

```diff
@@ -6,8 +6,8 @@ import { LapisSelector } from '../../data/LapisSelector';
 
 const usherUrl =
   'https://genome.ucsc.edu/cgi-bin/hgPhyloPlace?db=wuhCor1&phyloPlaceTree=hgPhyloPlaceData/wuhCor1' +
-  '/public.plusGisaid.latest.masked.pb&subtreeSize=1000&remoteFile=';
-const defaultOrderAndLimit: OrderAndLimitConfig = { orderBy: 'random', limit: 1000 };
+  '/public.plusGisaid.latest.masked.pb&subtreeSize=5000&remoteFile=';
+const defaultOrderAndLimit: OrderAndLimitConfig = { orderBy: 'random', limit: 5000 };
```

Comment:
Actually it's probably best to leave this at 1k, and just up the subtreeSize
It's a bit silly to peruse 5k sequences, I've never wanted to look at that many sequences at once.

What matters is having a lot of context, the more context the better.